lamenv_test.go

```diff
@@ -932,6 +968,37 @@ func TestMarshal(t *testing.T) {
 				"C",
 			},
 		},
+		{
+			title: "time.Duration",
+			conf: &struct {
+				A time.Duration
+				B *time.Duration
+				C time.Duration
+				D *time.Duration
+				E time.Duration  `mapstructure:"e,omitempty"`
+				F *time.Duration `mapstructure:"f,omitempty"`
+				G *time.Duration `mapstructure:"g,omitempty"`
+			}{
+				A: 1 * time.Minute,
+				B: &duration3h,
+				C: durationEmpty,
+				D: &durationEmpty,
+				E: durationEmpty,
+				F: &durationEmpty,
+				G: nil,
+			},
+			result: map[string]string{
+				"A": "1m0s",
+				"B": "3h0m0s",
```

Comment:
This can be odd to see 0m0s, I verified and yaml.v3 library is doing the same.

It is simply relying on the ``time.Duration`` implementation of ``String()`` method